apluslms_roman/settings.py

```diff
@@ -112,10 +112,11 @@ def update_from_namespace(self, namespace, store=False):
                     set_(option, value)
 
 
-class RomanSettings(ArgumentSettings):
+class GlobalSettings(ArgumentSettings):
     name = 'roman_settings'
     schema = name
     version = Version(1, 0)
+    _version_key = "version"
```

Comment:
This fixes something, so what was the problem? I have a feeling that there is a bug in the document metaclass, which would solve it..

Reply:
I mentioned this to you earlier, some stuff in the Document class requires version_key, but a Document can be created without a version_key

Reply:
I started to update `DocumentMeta` to fix this.

Reply:
I'm pretty sure this is now fixed in milestone0.3 in commit 30be357fc7e79ec50180f46b13295505ba79669b

Reply:
Can you test if this is still needed?